contact_updater/forms.py

```diff
@@ -0,0 +1,62 @@
+from django import forms
+
+PHONE_RE = r'^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]\d{3}[\s.-]\d{4}$'
```

Comment:
You re-assign this in the next line right? Or am I seeing things without coffee?